src/api/middleware/auth.test.js

```diff
@@ -0,0 +1,43 @@
+const createError = require('http-errors');
+const config = require('../../config');
+const Factory = require('../../tests/factories');
+const authMiddleware = require('./auth');
+
+const authTokenConfig = config.api.authToken;
+
+describe('authMiddleware', () => {
+  describe('when the token is not sent via headers', () => {
+    it('throws a 401 error', () => {
+      const ctx = Factory.build('koa-ctx', { headers: {} });
```

Comment:
Called `'koa-ctx'` and not `'ctx'` to avoid confusions with the bot's `'ctx'`

Reply:
multiple uses of same string, move to var